package.json

```diff
@@ -11,6 +11,9 @@
     "url": "jkimbo.com"
   },
   "files": ["docs/", "rules/", "processors/", "index.js"],
+  "engines": {
+    "node": ">= 6"
```

Comment:
Will this prevent users from installing the eslint-plugin-jest package on Node 4?

Reply:
oh oops, that was supposed to be 4. I just copied Jest's. Nice catch!

Reply:
851881186c21519e27426ae8f099bbe9bbf0837f